src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp/Exceptions/UnableToGetReleasesForPublicationException.cs

```diff
@@ -8,3 +8,11 @@ public class UnableToGetReleasesForPublicationException(string publicationSlug,
                  Unable to get releases for publication "{publicationSlug}". Error: "{errorMessage}"
                  """);
 
+/// <summary>
+/// Exception thrown when the call to the Content API to retrieve the release summary for a publication fails.
+/// </summary>
+public class UnableToGetReleaseSummaryForPublicationException(string publicationSlug, string releaseSlug, string? errorMessage)
+    : Exception($"""
+                 Unable to get release summary for release "{releaseSlug}" for publication "{publicationSlug}". Error: "{errorMessage}"
+                 """);
```

Comment:
When wouldn't there be an error message here? Looks like this could be mandatory and also avoids an situation where someone is looking at the exception message "Error: ", wondering where the error is.